packages/kit/types/config.d.ts

```diff
@@ -78,6 +78,13 @@ export type Config = {
 	preprocess?: any;
 };
 
+export type PrerenderErrorHandler = (errorDetails: {
+	status: number;
+	path: string;
+	parent: string | null;
```

Comment:
it's not clear to me from name alone - what is `parent`? should we rename to `referer`? would that be the same? or if the current name is the best, let's add some docs to explain what it is

Reply:
I think `referrer` is better, yeah.